privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -469,9 +479,16 @@ def clear(self):
     def audit_entry_to_dict(self, audit_entry):
         sig = None
         if self.sign_data:
-            sig = self.sign_object.verify(self._log_to_string(audit_entry),
-                                          audit_entry.signature,
-                                          self.verify_old_sig)
+            try:
+                sig = self.sign_object.verify(self._log_to_string(audit_entry),
+                                              audit_entry.signature,
+                                              self.verify_old_sig)
+            except UnicodeDecodeError as _e:
+                # TODO: Unless we trace and eliminate the broken unicode in the
+                #  audit_entry, we will get issues when packing the response.
+                log.warning('Could not verify log entry! We get invalid values '
+                            'from the database, please check the encoding.')
+                log.debug('{0!s'.format(traceback.format_exc()))
```

Comment:
There is a curly closing bracked missing.

Reply:
fixed in 7484253d